libs/packages/components/src/lib/new-popover/popover.ts

```diff
@@ -0,0 +1,403 @@
+    this.position = config.placement;
+    this.popperOptions = config.popperOptions;
+    this.triggers = config.triggers;
+    this.container = config.container;
+    this.disablePopover = config.disablePopover;
+    this.popoverClass = config.popoverClass;
+    this.openDelay = config.openDelay;
+    this.closeDelay = config.closeDelay;
+    this._positioning = sdsPositioning();
+    this._popupService = new PopupService<SdsPopoverWindow>(
+      SdsPopoverWindow,
+      injector,
+      viewContainerRef,
+      _renderer,
+      this._ngZone,
+      applicationRef
+    );
+  }
+
+  /**
+   * Opens the popover.
+   *
+   * This is considered to be a "manual" triggering.
+   * The `context` is an optional value to be injected into the popover template when it is created.
+   */
+  open(context?: any) {
+    if (!this._windowRef && !this._isDisabled()) {
+      // this type assertion is safe because otherwise _isDisabled would return true
+      const { windowRef, transition$ } = this._popupService.open(
+        this.sdsPopover as string | TemplateRef<any>,
+        context ?? this.popoverContext,
+        this.animation
+      );
+      this._windowRef = windowRef;
+      this._windowRef.setInput('animation', this.animation);
+      this._windowRef.setInput('title', this.sdsPopoverTitle);
+      this._windowRef.setInput('context', context ?? this.popoverContext);
+      this._windowRef.setInput('popoverClass', this.popoverClass);
+      this._windowRef.setInput('id', this._sdsPopoverWindowId);
+
+      this._renderer.setAttribute(this._getPositionTargetElement(), 'aria-describedby', this._sdsPopoverWindowId);
+
+      if (this.container === 'body') {
+        this._document.querySelector(this.container).appendChild(this._windowRef.location.nativeElement);
+      }
+
+      // We need to detect changes, because we don't know where .open() might be called from.
+      // Ex. opening popover from one of lifecycle hooks that run after the CD
+      // (say from ngAfterViewInit) will result in 'ExpressionHasChanged' exception
+      this._windowRef.changeDetectorRef.detectChanges();
+
+      // We need to mark for check, because popover won't work inside the OnPush component.
+      // Ex. when we use expression like `{{ popover.isOpen() : 'opened' : 'closed' }}`
+      // inside the template of an OnPush component and we change the popover from
+      // open -> closed, the expression in question won't be updated unless we explicitly
+      // mark the parent component to be checked.
+      this._windowRef.changeDetectorRef.markForCheck();
+
+      // Setting up popper and scheduling updates when zone is stable
+      this._ngZone.runOutsideAngular(() => {
+        this._positioning.createPopper({
+          hostElement: this._getPositionTargetElement(),
+          targetElement: this._windowRef!.location.nativeElement,
+          placement: this.position,
+          appendToBody: this.container === 'body',
+          baseClass: 'bs-popover',
```

Comment:
Can we change this to sds-popover? We could then rename the css classes as well.

Reply:
We can, but doing so will conflict with the existing popover/tooltip. I just tested it out and doing so caused the old popover/tooltip styles to be applied. 

If we remove the old styles, we can update these classes no problem, I just seem to remember we wanted to keep the old ones around from some reason.